components/layout.tsx

```diff
@@ -112,7 +112,7 @@ const TopNav: React.FC = () => {
                 style={{ alignContent: "flex-end" }}
                 onClick={closeNav}
               >
-                <NavLink href="/learnbasicsoftestimony">
+                <NavLink href="/learn/basics-of-testimony">
```

Comment:
Optional: Consider adding page URL's as constants in a routes.ts file or something, for ease of referencing

Reply:
I was about to before I realized that what we should (probably) do is separate the content from the views, either using a CMS or in files (either markdown or json). It would provide us with the routes and help us clean up some of the debt in the code.

Reply:
That's a good point. We want to support translations eventually, which requires that level of separation. I have only ever separated content for translation on mobile, so I'm interested in examples/patterns for the web. Maybe we can discuss next week.